inspirehep/config.py

```diff
@@ -307,14 +307,16 @@
 # SEARCH_ELASTIC_KEYWORD_MAPPING -- this variable holds a dictionary to map
 # invenio keywords to elasticsearch fields
 SEARCH_ELASTIC_KEYWORD_MAPPING = {
-    "author": ["authors.full_name"],
-    "exactauthor": ["exactauthor.raw"],
+    "author": ["authors.full_name", "authors.alternative_name"],
+    "exactauthor": ["exactauthor.raw", "authors.full_name",
+                    "authors.alternative_name"
+                    ],
```

Comment:
@Panos512 can you check how this plays with your branches inspirehep/invenio-search#17 and #715 ?